cnf-tests/testsuites/validationsuite/cluster/validation.go

```diff
@@ -408,4 +409,64 @@ var _ = Describe("validation", func() {
 			Expect(mcpExist).To(BeTrue(), fmt.Sprintf("was not able to find the xt_u32 machine config %s in the %s machine config pool", mc.Name, mcp.Name))
 		})
 	})
+
+	Context("gatekeeper mutation", func() {
+		It("should have the gatekeeper namespace", func() {
+			_, err := testclient.Client.Namespaces().Get(context.Background(), utils.GatekeeperNamespace, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+		})
+
+		It("should have the gatekeeper-operator-controller-manager deployment in running state", func() {
+			deploy, err := testclient.Client.Deployments(utils.GatekeeperNamespace).Get(context.Background(), utils.GatekeeperOperatorDeploymentName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(deploy.Status.Replicas).To(Equal(deploy.Status.ReadyReplicas))
+
+			pods, err := testclient.Client.Pods(utils.GatekeeperNamespace).List(context.Background(), metav1.ListOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			var operatorPods []corev1.Pod
+
+			for _, pod := range pods.Items {
```

Comment:
A better way would be to find the operator pod by label just like the others ... but it does not have any yet. Checking with Ivan if we can add something to identify the operator pod. Once we are able to do it, we can change this code.
(another way would be to check by ownerReferences)